src/main.rs

```diff
@@ -0,0 +1,18 @@
+pub mod api_client;
+pub mod error;
+pub mod serde;
+pub mod types;
```

Comment:
no need to `pub` here
```suggestion
mod api_client;
mod error;
mod serde;
mod types;
```

Reply:
also I don't think we need to declare any of these?

we only need to grab the `Client` here